ios/RNSScreen.mm

```diff
@@ -824,6 +1086,16 @@ - (UIView *)view
 
 @implementation RCTConvert (RNSScreen)
 
+RCT_ENUM_CONVERTER(
```

Comment:
Did it just work previously and then it stopped? Or what is the reason of adding it here?

Reply:
I used this convert method explicitly in `updateProps` method. Other solution I see is to change of incoming cpp type of `statusBarAnimation` prop from `std::string` to cpp enum - I can then add converting method in `RNSConvert`.